doc/source/Reference/ScriptingReference/CommonOperations/index.md

```diff
@@ -96,6 +96,23 @@ end = script['frameRange']['end'].getValue()
 script.context().setFrame( frame )
 ```
 
+Metadata
+--------
+
+### Register a value for a plug/node
+
+```
+Gaffer.Metadata.registerValue( plug, "name", value )
+Gaffer.Metadata.registerValue( node, "name", value )
+```
```

Comment:
Not common enough - this is the "code by example" section of the docs rather than an exhaustive API reference (which we still need to generate).